apps/back-office/src/app/(dashboard)/settings/profile-security/product/ProductSection.tsx

```
@@ -0,0 +1,261 @@
+  return (
+    <form
+      className="grid grid-cols-2 gap-4 items-center mb-3"
+      onSubmit={(e) => {
+        e.preventDefault();
+        e.stopPropagation();
+        form.handleSubmit();
+      }}
+    >
+      <form.Field
+        name="name"
+        children={(field) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)